versionedController/v1/email/email.go

```diff
@@ -45,18 +45,20 @@ func (emailController *EmailController) Post(c *gin.Context) {
 //Get verifies the email by a link
 func (emailController *EmailController) Get(c *gin.Context) {
 	token := c.Query("access")
+	redirectURL := types.PortalURL + "/verified-email"
 
 	jwtUserCredentials, err := controller.Server.GetUserFromToken(token)
 	if err != nil {
 		log.Error("Error occurred while parsing jwt token error: " + err.Error())
-		c.JSON(http.StatusUnauthorized, gin.H{
+		c.JSON(http.StatusInternalServerError, gin.H{
```

Comment:
Why this change?

Reply:
This change has been pushed by mistake. Will rectify it.